e2e/src/config/tests-config/setup.ts

```diff
@@ -267,6 +267,13 @@ export default class TestSetup {
     return this.config.L1.dummyContractAddress;
   }
 
+  public getLivenessSigner(): NonceManager {
+    // @WARNING: FOR LOCAL DEV ONLY - DO NOT REUSE THESE KEYS ELSEWHERE
+    const LIVENESS_SIGNER_PRIVATE_KEY = "0x234d87442cf7d43841fbe280febcdfabfb646added67bc19f7e42a5483f614c4";
```

Comment:
@VGau Any reason not to put this in e2e test config file `local.ts`, this would mean devnet and sepolia e2e tests (i.e. `test:e2e:dev` and `test:e2e:sepolia`) will use the same signer account when liveness test is enabled later on (for now its disabled), we can adjust it later, but just to check if there's any other reason to hardcoded it here